### ISSUE TITLE
Healthcheck Performance Testing

### ISSUE THREAD

**Issue by davidMcneil** [habitat-sh/habitat#7792]
Tests with the following configuration should be setup:

* Run a specified version of habitat
* Start 15 services consisting of: 
    * A simple `run` hook
    * A simple `healthcheck` hook
* The `healthcheck` hook interval should be configurable via an environment variable
* Metrics should be reported to datadog
    * Total CPU/memory usage
    * Per process CPU/memory usage
* The test should run on linux and windows

**Comment by mwrock** [comment]
Just want to point out that we should definitely use a bash hook for linux and not a single powershell hook that can be shared on both platforms. Otherwise we add extra bloat to the everyday linux use case.

**Comment by davidMcneil** [comment]
> we should definitely use a bash hook for linux and not a single powershell hook

Agreed, although it would be interesting to compare bash and powershell and powershell on linux and windows.
